Sources/Charcoal/Filters/FilterViewController.swift

```diff
@@ -125,7 +126,20 @@ public class FilterViewController: ScrollViewController, FilterBottomButtonViewD
 
 extension FilterViewController: UIGestureRecognizerDelegate {
     public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+        if let view = gestureRecognizer.view as? MKAnnotationView,
+            view.reuseIdentifier == MapPolygonFilterViewController.annotationId {
```

Comment:
Did you consider subclassing `MKAnnotationView` to a `PolygonPinAnnotationView` so you could check if it was an object of that type instead of looking at the `reuseIdentifier`?

Reply:
There is no polygon view, I just use `MKAnnotationView`. But I will do this instead: `view.annotation is PolygonSearchAnnotation`, so that I don't have to use the `reuseIdentifier` 👍